grakn-core/src/main/java/ai/grakn/concept/SchemaConcept.java

```diff
@@ -72,6 +72,12 @@
     SchemaConcept sup();
 
     /**
+     *
+     * @return All the the super-types of this
```

Comment:
`All the the super-types of this {@link SchemaConcept}`

Reply:
Modified all comments to fit this standard